crates/holochain/tests/tests/publish/mod.rs

```diff
@@ -29,56 +31,62 @@ async fn publish_terminates_after_receiving_required_validation_receipts() {
 
     let ((alice,), (bobbo,), (carol,), (danny,), (emma,), (fred,)) = apps.into_tuples();
 
+    let apps = [alice, bobbo, carol, danny, emma, fred];
+
+    for c in conductors.iter() {
+        c.holochain_p2p()
+            .test_set_full_arcs(apps[0].dna_hash().to_k2_space())
+            .await;
+    }
+
     let action_hash: ActionHash = conductors[0]
-        .call(&alice.zome(TestWasm::Create), "create_entry", ())
+        .call(&apps[0].zome(TestWasm::Create), "create_entry", ())
         .await;
 
-    // Wait until they all see the created entry, at that point validation receipts should be getting sent soon
-    await_consistency(60, [&alice, &bobbo, &carol, &danny, &emma, &fred])
-        .await
-        .unwrap();
+    tokio::time::timeout(
```

Comment:
Think it's possible to use the [`retry_until_timeout`](https://github.com/holochain/holochain/blob/feat/integrate-k2/crates/holochain/src/test_utils.rs#L68) that @jost-s recommended on MM here?

Reply:
Perhaps, but with my need for the double loop it's not an exact fit, and I like being able to see the components to make it clearer what is going on incase we need to adjust the timings in the future.